components/Ink/src/private/MDCLegacyInkLayer.m

```diff
@@ -165,6 +164,12 @@ - (CAMediaTimingFunction *)logDecelerateEasing {
                                                      0.157:(float)0.72:(float)0.386:(float)0.987];
 }
 
+- (void)animationDidStart:(CAAnimation *)anim {
+  if (!self.isAnimationCleared) {
```

Comment:
I'm not sure yet if we should actually calling back here every time although the animation was cleared. Especially in the `animationDidStop:` case where the `MDCLegacyInkLayer` actually is doing some internal work (removing foreground and background ripples from it's `foregroundRipples` and `backgroundRipples` array).